docs/docs/examples/guides/camera-calibration.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 9
+title: Camera calibration walkthrough
+---
+
+import cameraCalibration from './assets/camera-calibration.gif'
+
+# Camera calibration
+
+Accurate camera calibration is essential for ensuring that visual detections and measurements correspond correctly to
+real-world geometry. In many perception pipelines, critical output is expressed in image pixel coordinates. However, if
+your camera's lens introduces distortion or if the intrinsic parameters are unknown, these pixel coordinates can deviate
+significantly from their true positions.
+
+A calibration procedure allows you to determine your camera's intrinsic (focal length, optical center) and extrinsic
+(position, orientation) parameters, as well as its distortion coefficients. Once known, these parameters can be used to
+undistort images in real time, improving the spatial accuracy of detections and any downstream estimation tasks (e.g.,
+pose reconstruction, depth reasoning, or robot alignment).
+
+## AICA helpers for calibrating cameras
+
+In a typical ROS scenario, you may have a camera whose image is published as a `sensor_msgs::msg::Image` signal, a camera officially supported by an AICA collection release, or a USB
```

Comment:
This is more a question but do we anticipate that one has to calibrate Realsense and Orbbec cameras? If yes, then it's also not clear how one would apply this calibration, right?

Reply:
If one is not using the official drivers for those then they would potentially have to. Our existing components (for these 2) do output the signal for camera info already.

But in the case they do, which part isn't clear? The process doesn't change, but we don't explain how they'll get to have a `sensor_msgs::msg::Image` signal. Perhaps my use of "signal" is the culprit here. We can work with any topic of this kind coming from any sort of ROS 2 node.